examples/persistent-volumes/volumes/local-01.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolume
+apiVersion: v1beta3
+metadata:
+  name: pv0001
+  labels:
+    type: local
```

Comment:
I glad to receive the feedback and will correct the obvious and actionable stuff.  Words and terms are easily changed, too, if there is consensus on a term.  Many of them have already been put through the wringer.